core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorSplitManager.java

```diff
@@ -20,7 +20,6 @@ default ConnectorSplitSource getSplits(
             ConnectorTransactionHandle transaction,
```

Comment:
Can you remove this deprecated method now?

Reply:
My changes had nothing to do with that. The method is still used